### ISSUE TITLE
Feature request: general EEW, not only for Japan.

### ISSUE THREAD

**Issue by JaySNL** [Team-Fruit/EEWBot#1]
Adjusting the bot to use more locations, besides Japan, would increase the general use of the bot.
Perhaps set it with a filter for a specific location or just generally everywhere.

**Comment by sjcl** [comment]
Countries with EEW systems are limited. Outside Japan, it is located in Taiwan, Mexico, Turkey, Romania, etc.    
However, as there is no cohesion in the systems of those countries, I do not know whether that is possible.

**Comment by JaySNL** [comment]
Check out https://tool.siakbary.my.id/p/earthquake.html

The site uses a TTS approach to any notified / marked EQ. Maybe usefull to
add as a TTS feature?

**Comment by sjcl** [comment]
I found the API of this site, but I can't find the official document.

**Comment by sjcl** [comment]
https://api.siakbary.my.id/earthquake/geo.json

**Comment by sjcl** [comment]
Please let me know if I am misunderstanding.

**Comment by JaySNL** [comment]
If you could hook into the json or api (i asked the site owner to check this 'issue') you can have that be spoken or typed in TTS?

**Comment by sjcl** [comment]
It is possible to use TTS with Discord.

**Comment by JaySNL** [comment]
Yes, i meant into the current bot. Read out the JSON to use as a general notifier, added on top of the EEW.
